content/markdown/education/index.md

```diff
@@ -3,8 +3,8 @@ title: Education and Training
 layout: education/main.njk
 mainimage: https://cdn.tnris.org/images/keep_pace_banner.jpg
 smallimage: https://cdn.tnris.org/images/training_keep_pace_md.jpg
-abstract: Technical training courses bring the latest GIS technologies and skills to a range of professional skill levels in the workplace.
```

Comment:
abstract removed from front matter but still used in `templates/education/main.njk` on line 19?